docs/release-notes/15-4-0/README.md

```diff
@@ -10,18 +10,102 @@ release_date: 2025-03-06
 
 Release date: 2025-03-06
 
-We released OpenProject [OpenProject 15.4.0](https://community.openproject.org/versions/2165).
-The release contains several bug fixes and we recommend updating to the newest version.
-In these Release Notes, we will give an overview of important feature changes.
-At the end, you will find a complete list of all changes and bug fixes.
+We released OpenProject [OpenProject 15.4.0](https://community.openproject.org/versions/2165). The release contains several bug fixes, and we recommend updating to the newest version. In these Release Notes, we will give an overview of important feature changes. At the end, you will find a complete list of all changes and bug fixes.
 
 ## Important feature changes
 
-<!-- Inform about the major features in this section -->
+Take a look at the release video showing most important features introduced in 15.4.0:
+
+VIDEO
+
+### Enable automatic scheduling mode - to start as soon as possible
+
+Starting with OpenProject 15.4, users will be able to choose whether they want to manually schedule their work packages or enable automatic scheduling. This highly requested feature of automatic scheduling, based on predecessor/successor relations, will make managing complex project setups much easier.
+
+> [!NOTE]
+> The new feature will not overwrite existing dates for any existing work packages. However, it can change the scheduling mode. By default, manual scheduling is enabled. Existing work packages with children will be in automatic mode – with dates derived from their children.
+
+![New scheduling modes: Automatic and Manual](manualAutomatic_schedulingModes.png)
+
+In **manual mode**, users can freely select dates and can set timelines based on specific needs, deadlines or external factors. Manually scheduled work packages behave as they did before 15.4.
+
+In the **new automatic mode**, manually setting a start date is not possible. Instead, when a task is scheduled, the date picker automatically determines the start date based on the nearest predecessor. In other words: The work package starts as soon as possible, considering related work packages. However, you can still define a duration, which adjusts the finish date accordingly. This ensures a reliable sequence of automatically scheduled work packages that adapt dynamically to planning changes.
```

Comment:
Within the application, the term `closest predecessor` is used. Here, `nearest` is used. 

Mentioning using lag to determine the miniumum number of days between two work packages as well as non working days (at least name dropping) could answer some questions before they are asked.

Reply:
Not sure about it being necessary, but the "Single date mode" is not mentioned.